DependencyInjection/FOSHttpCacheExtension.php

```diff
@@ -88,7 +88,7 @@ public function load(array $configs, ContainerBuilder $container)
         if ($config['tags']['enabled']) {
             // true or auto
             $container->setParameter($this->getAlias().'.tag_handler.header', $config['tags']['header']);
-            $loader->load('tag_listener.xml');
+            $loader->load('tag.xml');
```

Comment:
i would opt to call the file cache_tagging.xml to be more explicit - there is several different kind of "tag" in symfony, e.g. service tags.